Source/WTF/wtf/URLParser.cpp

```diff
@@ -2887,12 +2887,12 @@ auto URLParser::parseURLEncodedForm(StringView input) -> URLEncodedForm
     for (StringView bytes : input.split('&')) {
         auto equalIndex = bytes.find('=');
         if (equalIndex == notFound) {
-            auto name = formURLDecode(bytes.toString().replace('+', 0x20));
+            auto name = formURLDecode(makeStringByReplacingAll(bytes.toString(), '+', 0x20));
```

Comment:
This call site needs the `StringView` version you mention.

Reply:
Yes, as I said in the change log, there are quite a few call sites that would benefit from that. I am planning to look into it in a follow-up. In the mean time, this doesn't make anything worse.